### PR TITLE
Add delete index option

### LINKED ISSUE #67: Add back "Delete index" feature

### PR DESCRIPTION
**Pull request by junyi73**
Closes #67

### CODE REVIEW

**Comment by t-leongchuan** [comment]
Commit messages are not conventional (2nd and 3rd commit to be specific)

**Comment by t-leongchuan** [comment]
Other than that the changes generally seem okay

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java [hunk 1]
```diff
@@ -55,6 +78,16 @@ public CommandResult execute(Model model) throws CommandException {
         return new CommandResult(String.format(MESSAGE_DELETE_PERSON_SUCCESS, Messages.format(personToDelete)));
     }
 
+    private CommandResult deleteByIndex(Model model) throws CommandException {
+        List<Person> lastShownList = model.getFilteredPersonList();
+        if (targetIndex.getZeroBased() >= lastShownList.size()) {
```
Comment (slidings) [inline src/main/java/seedu/address/logic/commands/DeleteCommand.java:83]: Using ```getOneBased() > lastShownList.size()``` may be more intuitive, but i see why zerobased was used for consistency on the next line

**Review by slidings — APPROVED** [review]

**Comment by slidings** [comment]
LGTM

**Pull request merged**